### PR TITLE
Update the dependencies of jmimemagic

### PR DESCRIPTION
**Pull request by madoar**
In addition this PR fixes the class name in the `Class.forName`.

### CODE REVIEW

**Comment by madoar** [comment]
We have an additional problem. It seems like the tests of `jmimemagic` fail because it misidentifies the mimetype of the (microsoft) office formats as `zip`:

```
Failed tests: 
  MagicTest.testOfficeOpenXmlPresentation:317 error in testOfficeOpenXmlPresentation(). message: expected:<application/[vnd.openxmlformats-officedocument.presentationml.presentation]> but was:<application/[zip]>
  MagicTest.testOfficeOpenXmlPresentation2:337 error in testOfficeOpenXmlPresentation2(). message: expected:<application/[vnd.openxmlformats-officedocument.presentationml.presentation]> but was:<application/[zip]>
  MagicTest.testOfficeOpenXmlDocument2:176 error in testOfficeOpenXmlDocument2(). message: expected:<application/[vnd.openxmlformats-officedocument.wordprocessingml.document]> but was:<application/[zip]>
  MagicTest.testOfficeOpenXmlWorkbook2:276 error in testOfficeOpenXmlWorkbook2(). message: expected:<application/[vnd.openxmlformats-officedocument.spreadsheetml.sheet]> but was:<application/[zip]>
  MagicTest.testOfficeOpenXmlWorkbook:256 error in testOfficeOpenXmlWorkbook(). message: expected:<application/[vnd.openxmlformats-officedocument.spreadsheetml.sheet]> but was:<application/[zip]>
  MagicTest.testOfficeOpenXmlDocument:156 error in testOfficeOpenXmlDocument(). message: expected:<application/[vnd.openxmlformats-officedocument.wordprocessingml.document]> but was:<application/[zip]>

```

Any ideas?

File: pom.xml [hunk 1]
```diff
@@ -1,321 +1,227 @@
...
-    </dependency>-->
-
-  </dependencies>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-  </properties>
-
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (plata) [inline pom.xml:2]: Is it a good idea to have these whitespace/formatting changes? Might give problems when updating to/from upstream.

**Comment by plata** [comment]
Regarding the test: I don't know how jmimemagic works, so I would have to debug the test to say anything as well.

File: pom.xml (hunk 1 above)
Comment (madoar) [reply pom.xml:2]: I would not count on any updates to the upstream. The latest change there is from 2017

**Comment by madoar** [comment]
https://github.com/arimus/jmimemagic/issues/30 seems to be linked to the issue I've stumbled over

**Comment by plata** [comment]
Then we shouldn't care about it.

**Comment by madoar** [comment]
It could block a jitpack build (but I'm not sure)

**Comment by madoar** [comment]
Ok the build should work (see https://jitpack.io/docs/BUILDING/#maven-projects).
Then I propose we merge this and open a new issue on this forked repository for the failing tests.

File: pom.xml (hunk 1 above)
Comment (plata) [reply pom.xml:2]: Ok.

**Review by plata — APPROVED** [review]

**Pull request merged**